app/components/search/EligibilitiesRefinementList.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connectRefinementList } from 'react-instantsearch/connectors';
+
+class EligibilitiesRefinementList extends Component {
+  static propTypes = {
+    items: PropTypes.array.isRequired,
+    refine: PropTypes.func.isRequired,
+    currentRefinement: PropTypes.array.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.changeRefinement = this.changeRefinement.bind(this);
+    this.setChecks = this.setChecks.bind(this);
+    this.eligibilitiesMapping = {
+      Disability: ['Disability', 'Developmental Disability', 'Physical Disability', 'Learning Disability', 'Intellectual Disability'],
```

Comment:
this is missing a couple of MOHCD's suggested eligibilities -- Immigrants and LGBTQ.

Reply:
added